From 62e40b00b2e7d1d6f5d9a3b91a9e97a797f7ed34 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Dec 2024 17:36:26 +0800
Subject: [PATCH 206/301] AOSCOS: platform: rs780e-acpi: deprecate
 pci_get_bus_and_slot()

The commit 5cf0c37a71da ("PCI: Remove pci_get_bus_and_slot() function")
removed pci_get_bus_and_slot() in favour of pci_get_domain_bus_and_slot(),
which added an extra first argument for setting the PCI domain (which
defaulted to 0 with pci_get_bus_and_slot().

Add 0 as the first parameter to complete the transition.

Fixes: "AOSCOS: MIPS: Loongson: Add ACPI Power Button driver"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/platform/mips/rs780e-acpi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/platform/mips/rs780e-acpi.c b/drivers/platform/mips/rs780e-acpi.c
index 35d2be41a54c..6e53355be9d9 100644
--- a/drivers/platform/mips/rs780e-acpi.c
+++ b/drivers/platform/mips/rs780e-acpi.c
@@ -95,7 +95,7 @@ static int __init power_button_init(void)
 	int ret;
 	struct pci_dev *dev;
 
-	dev = pci_get_bus_and_slot(0, 0);
+	dev = pci_get_domain_bus_and_slot(0, 0, 0);
 	switch (dev->vendor) {
 	case PCI_VENDOR_ID_AMD:
 	case PCI_VENDOR_ID_ATI:
-- 
2.48.1

